errai-demos/errai-security-demo/pom.xml

```diff
@@ -310,6 +311,14 @@
           <extraJvmArgs>-Xmx1G -XX:CompileThreshold=7000 -Derrai.jboss.home=${errai.jboss.home} -Derrai.dev.context=${errai.dev.context}</extraJvmArgs>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven.deploy.plugin.version}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
```

Comment:
I am not 100% sure this will pass productized build. IIRC skipping deployment fails the build there. @ryanzhang or @AntStephenson could you please clarify?